src/Workspaces/Core/Portable/Execution/Serializer_Asset.cs

```diff
@@ -30,6 +30,7 @@ public void SerializeSourceText(ITemporaryStorageWithName storage, SourceText te
             {
                 writer.WriteInt32((int)SerializationKinds.MemoryMapFile);
                 writer.WriteString(storage.Name);
+                writer.WriteInt64(storage.Offset);
```

Comment:
❓ Is there a serialization version or something that needs to change, or do we just use this?